mysql/populate.sql

```diff
@@ -0,0 +1,87 @@
+USE mintern;
+
+INSERT INTO Major (name) VALUES 
```

Comment:
If values inserted will not be in same line as `VALUES` keyword, then it needs to be in a newline.
Would look like this:
```
INSERT INTO Major (name)
VALUES
  ('ITC'),
  ('INT'),
  ('ISD'),
  ('ITI'),
  ('IMT');
```

Reply:
done